### ISSUE TITLE
Mathematical representation of positivity assumption for continuous exposures

### ISSUE THREAD

**Issue by LucyMcGowan** [LucyMcGowan/writing-positivity-continous-ps#6]
@bblette1 do you know of a convention for writing the positivity assumption for continuous exposures? Since the probability of having any value in a continuous distribution is 0 I assume it is something like: $f_{x|z} >0$ rather than $0 < P(X|Z) < 1$ ?

**Comment by bblette1** [comment]
Yeah that's right, in the past I've written it as assuming f(X = x | Z) > 0 for all x where dF(Z) > 0

**Comment by LucyMcGowan** [comment]
Nice, I’ll update that in the write up rather than the general form I think

**Comment by bblette1** [comment]
Sounds good. I haven't had time to contribute to the written sections, but hope to do so next week!
